Failing to go get wallet module

Hey. FYI I'm trying to go get the wallet module to use it, but getting this:

```
$ go get github.com/btcsuite/btcwallet/wallet
# github.com/btcsuite/btcwallet/wallet
/home/user/go/pkg/mod/github.com/btcsuite/btcwallet@v0.12.0/wallet/createtx.go:148:43: cannot use changeSource (type *txauthor.ChangeSource) as type txauthor.ChangeSource in argument to txauthor.NewUnsignedTransaction
/home/user/go/pkg/mod/github.com/btcsuite/btcwallet@v0.12.0/wallet/createtx.go:330:16: undefined: txsizes.NestedP2WPKHPkScriptSize
/home/user/go/pkg/mod/github.com/btcsuite/btcwallet@v0.12.0/wallet/createtx.go:358:42: invalid composite literal type txauthor.ChangeSource
/home/user/go/pkg/mod/github.com/btcsuite/btcwallet@v0.12.0/wallet/psbt.go:198:44: cannot use changeSource (type *txauthor.ChangeSource) as type txauthor.ChangeSource in argument to txauthor.NewUnsignedTransaction
```

Hey @guggero. How would I use it in my code?

I added `github.com/btcsuite/btcwallet/wallet` as an import, did `go mod tidy` and the wallet module was successfully found in btcwallet. But then I try to build:
```
$ go build -o main ./main.go
# github.com/btcsuite/btcwallet/wallet
/home/user/go/pkg/mod/github.com/btcsuite/btcwallet@v0.12.0/wallet/createtx.go:148:43: cannot use changeSource (type *txauthor.ChangeSource) as type txauthor.ChangeSource in argument to txauthor.NewUnsignedTransaction
/home/user/go/pkg/mod/github.com/btcsuite/btcwallet@v0.12.0/wallet/createtx.go:330:16: undefined: txsizes.NestedP2WPKHPkScriptSize
/home/user/go/pkg/mod/github.com/btcsuite/btcwallet@v0.12.0/wallet/createtx.go:358:42: invalid composite literal type txauthor.ChangeSource
/home/user/go/pkg/mod/github.com/btcsuite/btcwallet@v0.12.0/wallet/psbt.go:198:44: cannot use changeSource (type *txauthor.ChangeSource) as type txauthor.ChangeSource in argument to txauthor.NewUnsignedTransaction
```

--- a/go.mod
+++ b/go.mod
@@ -5,9 +5,9 @@ require (
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
 	github.com/btcsuite/btcutil/psbt v1.0.3-0.20201208143702-a53e38424cce
-	github.com/btcsuite/btcwallet/wallet/txauthor v1.0.0
+	github.com/btcsuite/btcwallet/wallet/txauthor v1.0.1-0.20210329233242-e0607006dce6
 	github.com/btcsuite/btcwallet/wallet/txrules v1.0.0
-	github.com/btcsuite/btcwallet/wallet/txsizes v1.0.0
+	github.com/btcsuite/btcwallet/wallet/txsizes v1.0.1-0.20210519225359-6ab9b615576f
 	github.com/btcsuite/btcwallet/walletdb v1.3.5
 	github.com/btcsuite/btcwallet/wtxmgr v1.3.0
 	github.com/btcsuite/websocket v0.0.0-20150119174127-31079b680792

